app/components/gcse_requirements_component.rb

```diff
@@ -0,0 +1,50 @@
+class GcseRequirementsComponent < ViewComponent::Base
+  attr_reader :course
+
+  def initialize(course:)
+    @course = course
+  end
+
+private
+
+  def required_gcse_content(course)
+    case course.level
+    when "primary"
+      "Grade 4 (C) or above in English, maths and science, or equivalent qualification"
+    when "secondary"
+      "Grade 4 (C) or above in English and maths, or equivalent qualification"
+    end
+  end
+
+  def pending_gcse_content(course)
+    if course.accept_pending_gcse.present?
```

Comment:
```suggestion
    if course.accept_pending_gcse?
```

Reply:
Hey so  `.accept_pending_gcse?` doesn't currently exist on the course model. Is it worth adding for this?

I can achieve the same result with `course.accept_pending_gcse`

Reply:
Rails will create that method for you, so it's already available to use. Same with the other boolean attributes that we're checking in the `equivalencies` method below.

Reply:
I did try this initially but it doesn't seem to pull through:
<img width="1790" alt="Screenshot 2021-06-30 at 13 08 01" src="https://user-images.githubusercontent.com/62567622/123957891-46c34b80-d9a4-11eb-910a-06a6e528475a.png">

Reply:
from what I've seen on publish the convention seems to be to just go with `course.accept_pending_gcse`

Reply:
Ah, my fault! I'm working in TTAPI at the moment where these are actual ActiveRecord objects. +1 to using `course.accept_pending_gcse` 👍🏼

Reply:
Awesome! Thanks Malcolm 🎉